pandas/io/tests/test_packers.py

```diff
@@ -496,6 +517,46 @@ def test_compression_blosc(self):
             assert_frame_equal(self.frame[k], i_rec[k])
 
 
+class TestMsgpack():
```

Comment:
I think it doesn't. I can make this particular test class fail by removing the data files.  Inheriting makes the `yield` syntax for testing fail. This is how pickle is tested too.